### PR TITLE
Use SWC for better CI/test/build performance

### PR DESCRIPTION
**Pull request by andersfischernielsen**
This ensures that the `.swcrc` configuration is present when building container images for distribution. This speeds up builds massively (200-300% depending on CPU). 

Using [SWC](https://swc.rs/) globally for builds also means that we don't need a separate SWC task for CI and testing. These have therefore been removed. 

The resulting container image has been tested locally with expected secrets and runs as expected.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -4,6 +4,7 @@ WORKDIR /usr/src/app
 
 COPY yarn.lock package.json tsconfig.json .
 COPY src/ ./src
+COPY .swcrc ./.swcrc
```
Comment (erwinw) [inline Dockerfile:7]: Smart.

**Review by erwinw — APPROVED** [review]
LGTM

File: Dockerfile (hunk 1 above)
Comment (andersfischernielsen) [reply Dockerfile:7]: Haha, I know 😮‍💨

**Review by EliasJorgensen — APPROVED** [review]
I
AM
SPEED
🚗

**Pull request merged**